Yosemite/Yosemite/Model/Mocks/ActionHandlers/MockAppSettingsActionHandler.swift

```diff
@@ -9,23 +9,27 @@ struct MockAppSettingsActionHandler: MockActionHandler {
 
     func handle(action: ActionType) {
         switch action {
-            case .loadInitialStatsVersionToShow(let siteId, let onCompletion):
-                loadInitialStatsVersionToShow(siteId: siteId, onCompletion: onCompletion)
-            case .setStatsVersionLastShown:
-                // This case needs to be handled to avoid crashing when running screenshots
-                // Once the enum is removed, this can be as well.
-                success()
-            case .loadFeedbackVisibility(let type, let onCompletion):
-                loadFeedbackVisibility(type: type, onCompletion: onCompletion)
-            case .setInstallationDateIfNecessary(let date, let onCompletion):
-                setInstallationDateIfNecessary(date: date, onCompletion: onCompletion)
-            case .loadProductsSettings(let siteId, let onCompletion):
-                loadProductSettings(siteId: siteId, onCompletion: onCompletion)
-            case .loadEligibilityErrorInfo(let onCompletion):
-                onCompletion(.failure(AppSettingsStoreErrors.noEligibilityErrorInfo))
-            case .loadOrderAddOnsSwitchState(let onCompletion):
-                onCompletion(.failure(AppSettingsStoreErrors.noEligibilityErrorInfo))
-            case .resetEligibilityErrorInfo:
+        case .loadInitialStatsVersionToShow(let siteId, let onCompletion):
+            loadInitialStatsVersionToShow(siteId: siteId, onCompletion: onCompletion)
+        case .setStatsVersionLastShown:
+            // This case needs to be handled to avoid crashing when running screenshots
+            // Once the enum is removed, this can be as well.
```

Comment:
This comment actually predates this PR, it's just indented differently here 😅 now that I was reading it, I deleted this comment in [TODO](https://github.com/woocommerce/woocommerce-ios/pull/5936/commits/c6b6fa92a96ebf65d625a83b9be49382dc86be20) since we only implement necessary actions and this case is one of them